src/components/HierarchyParents.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { ExpandableItemsProps } from "./ExpandableItems";
+import { TermAccordion } from "./TermAccordion";
+import { NormalEnd, ParentEnd, ParentsEnd } from "./HierarchyItem";
+
+const HierarchyParents: React.FC<ExpandableItemsProps> = (props) => {
+  if (props.items?.length === 1)
+    return (
+      <TermAccordion
+        term={props.items[0]}
+        level={props.level}
+        connector={<NormalEnd />}
+      />
+    );
+  //If there is more than one parent, connectors need to look differently
+  return (
+    <>
+      {props.items?.map((item, index) => {
+        if (index + 1 === props.items.length)
+          return (
+            <TermAccordion
+              level={props.level}
+              term={item}
+              key={item.uri}
+              connector={<ParentsEnd />}
```

Comment:
If the return statements differ only in one prop, then it is possible to make that prop conditional, or better yet prepare the connector type to an extra variable like this:
```typescript
const connector = index + 1 === props.items.length ? <ParentsEnd /> : <ParentEnd />
return (
  <TermAccordion
    ...
    connector={connector}
  />
);
```